processing/src/main/java/io/druid/query/aggregation/LongMaxAggregatorFactory.java

```diff
@@ -22,48 +22,27 @@
 import com.fasterxml.jackson.annotation.JacksonInject;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.Preconditions;
 import io.druid.java.util.common.StringUtils;
 import io.druid.math.expr.ExprMacroTable;
-import io.druid.math.expr.Parser;
-import io.druid.segment.BaseLongColumnValueSelector;
 import io.druid.segment.ColumnSelectorFactory;
-import io.druid.segment.ColumnValueSelector;
 
 import java.nio.ByteBuffer;
 import java.util.Arrays;
-import java.util.Collections;
-import java.util.Comparator;
 import java.util.List;
-import java.util.Objects;
 
 /**
  */
-public class LongMaxAggregatorFactory extends AggregatorFactory
+public class LongMaxAggregatorFactory extends SimpleLongAggregatorFactory
```

Comment:
Please make `LongMaxAggregatorFactory`, `DoubleMaxAggregatorFactory` and `FloatMaxAggregatorFactory` identical except the primitive type and class name. Currently there are some pairwise differences between them: method order, constructor parameter order, the list of overridden methods, utility method usage, etc.

Reply:
done.